src/sidebar/components/excerpt.js

```diff
@@ -1,178 +1,193 @@
+        />
+      )}
+    </div>
   );
-  window.addEventListener('resize', overflowMonitor.check);
-  $scope.$on('$destroy', function() {
-    window.removeEventListener('resize', overflowMonitor.check);
-  });
-
-  // Watch for changes to the visibility of the excerpt.
-  // Unfortunately there is no DOM API for this, so we rely on a digest
-  // being triggered after the visibility changes.
-  $scope.$watch(isElementHidden, function(hidden) {
-    if (!hidden) {
-      overflowMonitor.check();
-    }
-  });
-
-  // Watch input properties which may affect the overflow state
-  $scope.$watch('vm.contentData', overflowMonitor.check);
-
-  // Trigger an initial calculation of the overflow state.
-  //
-  // This is performed asynchronously so that the content of the <excerpt>
-  // has settled - ie. all Angular directives have been fully applied and
-  // the DOM has stopped changing. This may take several $digest cycles.
-  overflowMonitor.check();
 }
 
-/**
- * @description This component truncates the height of its contents to a
- *              specified number of lines and provides controls for expanding
- *              and collapsing the resulting truncated element.
- */
-module.exports = {
-  controller: ExcerptController,
-  controllerAs: 'vm',
-  bindings: {
-    /** Whether or not expansion should be animated. Defaults to true. */
-    animate: '<?',
-    /**
-     * The data which is used to generate the excerpt's content.
-     * When this changes, the excerpt will recompute whether the content
-     * is overflowing.
-     */
-    contentData: '<',
-    /**
-     * Specifies whether controls to expand and collapse
-     * the excerpt should be shown inside the <excerpt> component.
-     * If false, external controls can expand/collapse the excerpt by
-     * setting the 'collapse' property.
-     */
-    inlineControls: '<',
-    /** Sets whether or not the excerpt is collapsed. */
-    collapse: '=?',
-    /**
-     * Called when the collapsibility of the excerpt (that is, whether or
-     * not the content height exceeds the collapsed height), changes.
-     *
-     * Note: This function is *not* called from inside a digest cycle,
-     * the caller is responsible for triggering any necessary digests.
-     */
-    onCollapsibleChanged: '&?',
-    /** The height of this container in pixels when collapsed.
-     */
-    collapsedHeight: '<',
-    /**
-     * The number of pixels by which the height of the excerpt's content
-     * must extend beyond the collapsed height in order for truncation to
-     * be activated. This prevents the 'More' link from being shown to expand
-     * the excerpt if it has only been truncated by a very small amount, such
-     * that expanding the excerpt would reveal no extra lines of text.
-     */
-    overflowHysteresis: '<?',
-  },
-  transclude: true,
-  template: require('../templates/excerpt.html'),
+Excerpt.propTypes = {
+  /**
+   * The content to render inside the container.
```

Comment:
I feel like we could probably get away with not commenting `children` props if we didn't feel like it. That said, I have no aversion to this comment.

Reply:
I agree. I'll remove it.